### PR TITLE
Update executor support error log instead of fail only

### PR DESCRIPTION
**Pull request by tengattack**
Fix executor run command failed but only rpc error status return & store.

Previous log like this:
```
rpc error: code = Unknown desc = exit status 1
```

After patching this PR, we have detail log:
```
exit status 1
firstline
```

related #438

### CODE REVIEW

**Comment by tengattack** [comment]
It seems CI failed because docker failed:
```
Status: Downloaded newer image for quay.io/coreos/etcd:latest
bec49e08b180d3d602cfca086d5085f7ae0be0c5998b9c264c1cc55939a163d5
Unable to find image 'dkron:latest' locally
docker: Error response from daemon: pull access denied for dkron, repository does not exist or may require 'docker login'.
See 'docker run --help'.
make: *** [test] Error 125
```

**Comment by victorcoder** [comment]
Please also rebase with master

**Comment by tengattack** [comment]
OK!

File: proto/executor.proto [hunk 1]
```diff
@@ -8,10 +8,11 @@ message ExecuteRequest {
   map<string, string> config = 2;
 }
 
-message ExecuteResponse {
+message ExecutorResponse {
```
[the remark this answers is not in the document]
Comment (tengattack) [reply proto/executor.proto]: image
Ummmm, did you mean this previously? Or I make a misunderstanding?

[the remark this answers is not in the document]
Comment (tengattack) [reply proto/executor.proto]: You mean I should change them both?

[the remark this answers is not in the document]
Comment (tengattack) [reply proto/executor.proto]: Or I could rollback the previous refactor~

[the remark this answers is not in the document]
Comment (victorcoder) [reply proto/executor.proto]: Sorry, my mistake! just revert the previous refactor. My first comment https://github.com/victorcoder/dkron/pull/448#pullrequestreview-165355105 was actually wrong. Thanks!

[the remark this answers is not in the document]
Comment (tengattack) [reply proto/executor.proto]: OK I will do it today

**Comment by tengattack** [comment]
I will do force push here, wait a minute

**Comment by tengattack** [comment]
Done~

**Comment by victorcoder** [comment]
Breaking change in this PR for plugins but much better error handling, thanks @tengattack

**Pull request merged**